packages/fetchai/skills/error/README.md

```diff
@@ -0,0 +1,9 @@
+# Error
+
+## Description
+
+The error skill is used to handle incoming envelopes which cannot be properly decoded or have no target skill.
```

Comment:
Maybe more descriptive like this:

The error skill is used to handle incoming envelops that contain error messages using the default protocol. 

Assume A sends a problematic message to B (e.g. the message cannot be decoded, has no target skill, etc). Then B can send an error message using the default protocol to A describing the problem. If A has the `error` skill, it can interpret the error message and take further actions, for example to correct the message and resend it to B.